docs/DevelopmentPlan/DevelopmentPlan.md

```diff
@@ -64,12 +64,22 @@ will be recorded by the capstone team.
 
 # 6 Team Member Roles
 
-[\[Will meetings be virtual? At least some meetings should likely be
-in-person. ---SS\]]{style="color: blue"}
+Meeting Chair: This position will coordinate meetings, ensure an agenda is set, 
+and that all parties are prepared for the meeting.
 
-[\[How will the meetings be structured? There should be a chair for all
-meetings. There should be an agenda for all meetings.
----SS\]]{style="color: blue"}
+Notetaker: They are responsible for the recording of meeting minutes and 
+ensuring that they are entered into GitHub.
+
+Project Manager: Responsible for ensuring that all team members are up to date 
+on work being performed and that action items for needed tasks are completed.
+
+Team Liaison: Mitchell is the team liaison. They are responsible for 
+maintaining  communications between the McMaster instructional team and the 
+City for the group.
+
+Reviewer: This role is responsible for reviewing other work and ensuring the 
+quality is at a required standard. This includes reviewing commits and pull 
+requests.
```

Comment:
I think it would make sense to add a "developer" role that has the responsibilities of writing, testing, and handling prs/reviewing. Also might make sense to add another role thats more technical like "devops lead" whos role is to maintain infrastructrue such as github actions, docker, aws services etc

Reply:
right now there isn't anything technical in the roles defined and looking at the slide: https://gitlab.cas.mcmaster.ca/fanr13/capstone/-/blob/main/Lectures/L02b_POCAndDevPlan/POCAndDevPlan.pdf?ref_type=heads it seems like there should be some mention of technical roles here